curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/converts-an-image-from-one-color-space-to-another.md

```diff
@@ -1,27 +1,22 @@
 ---
 id: 5d383f6f2cd33e8211fad377
-title: Enter in openCV
+title: Converts an image from one color space to another
 challengeType: 0
 videoUrl: 
 ---
 
 ## Description
 <section id='description'>
-Welcome to freeCodeCamp's HTML coding challenges. These will walk you through web development step-by-step.
-First, you'll start by building a simple web page using HTML. You can edit <code>code</code> in your <code>code editor</code>, which is embedded into this web page.
-Do you see the code in your code editor that says <code>&#60;h1&#62;Hello&#60;/h1&#62;</code>? That's an HTML <code>element</code>.
-Most HTML elements have an <code>opening tag</code> and a <code>closing tag</code>.
-Opening tags look like this:
-<code>&#60;h1&#62;</code>
-Closing tags look like this:
-<code>&#60;/h1&#62;</code>
-The only difference between opening and closing tags is the forward slash after the opening bracket of a closing tag.
-Each challenge has tests you can run at any time by clicking the "Run tests" button. When you pass all tests, you'll be prompted to submit your solution and go to the next coding challenge.
+We use the function: <code>cv.cvtColor (src, dst, code, dstCn = 0) </code>
+There are more than 150 color-space conversion methods available in OpenCV.js. But we will look into the most widely used one: RGBA to Gray.
```

Comment:
~~But we will look into~~ In this lesson we will look into

Reply:
Please briefly describe for which purpose it might be used. In example, grayscale image consists of a single color that means total size of image in bytes is 3 times less than RGB.

Reply:
It will be done!